src/bionics.cpp

```diff
@@ -1075,10 +1075,19 @@ static bool attempt_recharge( Character &p, bionic &bio, units::energy &amount,
 void Character::process_bionic( int b )
 {
     bionic &bio = ( *my_bionics )[b];
-    if( calendar::once_every( 1_hours ) && !bio.id->fuel_opts.empty() &&
-        bio.has_flag( "AUTO_START_ON" ) && power_level <= 0_mJ ) {
-        g->u.activate_bionic( b );
+
+    if( !bio.id->fuel_opts.empty() && bio.has_var( "AUTO_START" ) ) {
```

Comment:
Why don't you just make AUTO_START a normal member (`int` or something)?

Reply:
We can use var to implement other functions, while adding a new member would just be usefull for this functionality.

Reply:
Well, "to implement other functions" - other functions can just add their own members. Instead of adding untyped values to the map (which must be serialised/deserialised upon each use) we can use the C++ feature of typed class member.

That would make the code simpler and avoid potential for errors.

Reply:
Done